docs/usage/models.md

```diff
@@ -1133,7 +1133,8 @@ print(Model(a=3.000, b='2.72', c=b'binary data').model_dump())
 This is a deliberate decision of *pydantic*, and in general it's the most useful approach. See
 [here](https://github.com/pydantic/pydantic/issues/578) for a longer discussion on the subject.
 
-Nevertheless, [strict type checking](types/types.md#strict-types) is partially supported.
+Nevertheless, [strict type checking](types/standard_types.md#strict-types) is partially supported.
+[TODO: Discuss strict mode, but also retain the previous link to types/standard_types.md#strict-types]
```

Comment:
I noticed this needs to be addressed here, so wanted to create a note, but there is a separate issue for working on this file and didn't want to get distracted by fixing that now..